templates/index.html

```diff
@@ -108,7 +108,9 @@ <h2 class="fs-2">College of Engineering, Pune, India</h2>
           <div class="skew-down"></div>
           <div class="circle">Sprints</div>
           <div class="b-f content">
-            Development sprints are a key part of PyCon, and a chance for the contributors to open-source projects to get together face-to-face for up to two days of intensive learning, development and camaraderie.
+            Development sprints are a key part of PyCon.
+            A chance for the open-source contributors, evangelists, and newbies
+            to meet on-site for up to two days of intensive learning, development, and camaraderie.
```

Comment:
I am trying to include newbies in the dev sprints.
"Development sprints are a key part of PyCon, and a chance for the contributors to open-source projects to get together face-to-face for up to four days of intensive learning, development and camaraderie. Newbies sit with gurus, go out for lunch and dinner together, and have a great time while advancing their project."- This is the complete description of what Pycon US says. We need to find a way to include a sentence about new contributors as well :)